src/cephadm/cephadm

```diff
@@ -59,6 +59,10 @@ import tempfile
 import time
 import errno
 import struct
+from socketserver import ThreadingMixIn
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import signal
```

Comment:
Yes this is using the stdlib features, and yes it's basic...but I don't think we need complicated.

pex would bring it's own challenges. We don't know what that looks like, and that's even outside of the refactor work that would be needed. I'd vote for the keep it simple, until we need complicated.

Reply:
+1 to keep it simple until we need more complicated things. I thing we do not need a high performance http stack. Probably with the standard server from stdlib it will be enough unless we found any security problem.